.github/workflows/test.yml

```diff
@@ -9,11 +9,10 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@dcd71f646680f2efd8db4afa5ad64fdcba30e748
-      - name: Setup NodeJS
+      - uses: actions/checkout@3
```

Comment:
Could you undo those changes? Checkout is already at version 3, that's what the commit sha points to (https://github.com/actions/checkout/commit/dcd71f646680f2efd8db4afa5ad64fdcba30e748). Dependabot takes care of updating it in this repository. I also don't see a reason to remove the step name 🤔

Reply:
That was an oversight. My apologies for that 🙇 I am fixing it in my branch, and I'll update the PR in a bit. 🙏